src/vs/workbench/api/node/extHost.api.impl.ts

```diff
@@ -230,7 +230,10 @@ export function createApiFactory(
 			},
 			getCommands(filterInternal: boolean = false): Thenable<string[]> {
 				return extHostCommands.getCommands(filterInternal);
-			}
+			},
+			onDidExecuteCommand: (listener, thisArgs?, disposables?) => {
+				return extHostCommands.onDidExecuteCommand(listener, thisArgs, disposables);
```

Comment:
Check if proposed API is allows for this extensions, see: https://github.com/Microsoft/vscode/blob/84b1474d097789b7915445688b2201f9d897d0cf/src/vs/workbench/api/node/extHost.api.impl.ts#L501

Reply:
Added.